AuScope/AuScope-Portal-UI
# AUS-3948 Fixed

Fixed an issue where javascript plotting libraries were being broken by the angular `buldOptmizer` this bug was only affecting the audev build.

Steps to test:

- `npm run build-dev`
- `npm install http-server -g` # If not already installed
- `http-server dist/`

Then follow instructions on https://jira.csiro.au/projects/AUS/issues/AUS-3948

If you see the plot then it's all fixed :)

Also verify using the prod build 
- `npm run build-prod`
- `http-server dist/`
